### PR TITLE
Show RAM use and data rate when hovering an entity in stream view

### PR DESCRIPTION
**Pull request by emilk**
* Closes https://github.com/rerun-io/rerun/issues/2435

image

image

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested [demo.rerun.io](https://demo.rerun.io/pr/2997) (if applicable)

- [PR Build Summary](https://build.rerun.io/pr/2997)
- [Docs preview](https://rerun.io/preview/pr%3Aemilk%2Fshow-data-volume/docs)
- [Examples preview](https://rerun.io/preview/pr%3Aemilk%2Fshow-data-volume/examples)

### CODE REVIEW

**Comment by Wumpf** [comment]
pretty cool to have this, but are we sure we want to display it that prominently in the tooltip? 🤔

**Comment by emilk** [comment]
> pretty cool to have this, but are we sure we want to display it that prominently in the tooltip? 🤔

I'm not quite sure where else to show it. In the selection menu we always show the _current_ state of an entity (i.e. at the currently selected time), so adding a "lifetime" stat like this there felt out-of-place

File: crates/re_data_ui/src/item_ui.rs [hunk 1]
```diff
@@ -105,12 +105,86 @@ pub fn instance_path_button_to(
...
+
+    if stats.time_range.min < stats.time_range.max && 1 < num_events {
+        // Estimate a data rate.
+        //
+        // Let's do our best to avoid fencepost errors.
+        // If we log 1 MiB every second, then after three
+        // events we have a span of 2 seconds, and 3 MiB,
+        // but the data rate is still 1 MiB/s.
+        //
+        //          <-----2 sec----->
```
Comment (Wumpf) [inline crates/re_data_ui/src/item_ui.rs:148]: useful illustration, nice 👍

**Review by Wumpf — APPROVED** [review]
lgtm

But I don't understand why this doesn't show a rate on the web demo:
image

**Comment by emilk** [comment]
@Wumpf old build - it works now!

**Pull request merged**